src/signalrclient/hub_connection_impl.cpp

```diff
@@ -192,6 +190,78 @@ namespace signalr
         Close,
     };
 
+    signalr::value createValue(const web::json::value& v)
```

Comment:
These should probably live as statics on signalr_value

Reply:
I don't think so. My next PR will be moving them into json_hub_protocol.cpp. They are not for end user consumption.